.github/workflows/render-stan-docs.yaml

```
@@ -28,14 +28,14 @@ jobs:
         # Path to Doxyfile
         doxyfile-path: "Doxyfile"
         # Working directory
-        working-directory: "./inst/stan/functions/"
+        working-directory: "./inst/stan/stan_docs/"
```

Z
```suggestion
        working-directory: "./inst/stan/docs/"
```

Its in a folder called stan already so I think its duplicative.

---

Makes sense 😆